### ISSUE TITLE
Any possibility of supporting `transpose` operation?

### ISSUE THREAD

**Issue by ppershing** [google-coral/edgetpu#193]
According to https://coral.ai/docs/edgetpu/models-intro/#supported-operations coral does not seem to support Transpose op. I would presume that under some limiting circumstances (e.g., not transposing batch dim, etc.) the device should be able to support this. And the operation would be very helpful -- given size limitations, using grouped convolutions might be an easy way to get better accuracy. However, such architectures usually involve a channel "shuffle" so that channels from different groups can get mixed and such shuffle is usually implemented by reshape + transpose + reshape.

**Comment by ofeknesher2** [comment]
+1

**Comment by Namburger** [comment]
Hi guys, I do have a feature request opened for this internally, will keep update if we move forward with it.

**Comment by ofeknesher2** [comment]
For any one face this issue
by converting pytorch to **keras** (not tensorflow) i was able to fully convert my model to NHWC and by that avoiding the TRANSPOE nodes, edge tpu comile was able to map all operations to TPU  (:

**Comment by MekhailS** [comment]
+1, badly need this feature

**Comment by manoj7410** [comment]
Compiler version 16 supports 'Transpose' operations. Please see the list of supported operations at https://coral.ai/docs/edgetpu/models-intro/#supported-operations.
